src/modules/cssTweaks.js

```diff
@@ -25,6 +28,24 @@ const DEFAULT_SETTINGS = {
 			type: SETTING_TYPES.BOOLEAN,
 			value: false
 		},
+		[SETTING_IDS.HIDE_LEFT_SIDEBAR_BUTTON]: {
+			name: 'Hide the "< Show" button',
+			description: 'Make the little floating thing disappear',
+			type: SETTING_TYPES.BOOLEAN,
+			value: true
+		},
+		[SETTING_IDS.NORMALIZE_CHAT_FONT]: {
+			name: 'Normalize chat font',
+			description: "Make the font in premium accounts' chat messages the same size as others",
+			type: SETTING_TYPES.BOOLEAN,
+			value: true
+		},
+		[SETTING_IDS.HIDE_TWITTER_PROMPT]: {
+			name: 'Hide the Twitter-like prompt',
```

Comment:
Please rename this to `Hide the 'Post your thoughts' prompt" and swap the variable name to `HIDE_POST_YOUR_THOUGHTS`
Since the side does have actual links to twitter I found this confusing until I read the description.